### ISSUE TITLE
Hide the placehoder from the dropdown choices list 

### ISSUE THREAD

**Issue by HectorLS** [Choices-js/Choices#843]
```js
"choices.js": "^7.0.0"
```
Hello!

## Case scenario:
Common behaviour where the placeholder is not able to be selected from the dropdown list:
- placeholder should only be visible on page load
- after user select something else, it won't be able to select the placeholder back again

Base on the rule that  placeholder value is an empty string -> ```placeholder value = ''```

Is there any feature about this ? like `placeholderSelectable: true/false` or some idea to achieve this easily 🤔 ? 

Thanks in advance

**Comment by ksouthworth** [comment]
I'm also dealing with this issue, does anyone have a solution or a workaround?

**Comment by veacks** [comment]
I have found the following simple CSS workaround

```css
.choices__list--dropdown .choices__placeholder {
  display: none;
}
```

**Comment by Xon** [comment]
Implemented as part of #1166
